tests/TKSetupSamples.m

```diff
@@ -0,0 +1,36 @@
+//
+//  TKSetupSamples.m
+//  TokenSdkTests
+//
+//  Created by Larry Hosken on 10/25/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "TKTestBase.h"
+
+#import "TokenIOSync.h"
+#import "TokenIOBuilder.h"
+
+#import "TKUtil.h"
+
+@interface TKSetupSamples : TKTestBase
+
+@end
+
+@implementation TKSetupSamples
+
+- (void)testCreateSDKClient {
+    [self run: ^(TokenIOSync *tIO) {
+        TokenIOBuilder *builder = [TokenIOSync builder];
```

Comment:
Since this is a sample, it is better to alloc/init the object by TokenIOBuilder instead.
like TokenIOBuilder *builder = [TokenIOBuilder sandboxBuilder];

Reply:
Excellent, done.